bugsnag-android-core/src/main/java/com/bugsnag/android/Session.java

```diff
@@ -198,16 +198,20 @@ Session incrementUnhandledAndCopy() {
         return copySession(this);
     }
 
-    AtomicBoolean isTracked() {
-        return tracked;
+    boolean markTracked() {
+        return tracked.compareAndSet(false, true);
+    }
+
+    boolean markPaused() {
+        return isPaused.compareAndSet(true, false);
```

Comment:
Given that this are now encapsulated, can we change the visibility of the `isPaused` to `private` instead of package-protected?

Reply:
`isPaused` in `Session` can not be private since it is used and reset in `SessionTracker`